gensim/models/callbacks.py

```diff
@@ -18,63 +18,84 @@
 
 
 class Metric(object):
-    """
-    Base Metric class for topic model evaluation metrics
+    """Base Metric class for topic model evaluation metrics.
+
+    Concrete implementations include:
+
+        * :class:`~gensim.models.callbacks.CoherenceMetric`
+        * :class:`~gensim.models.callbacks.PerplexityMetric`
+        * :class:`~gensim.models.callbacks.DiffMetric`
+        * :class:`~gensim.models.callbacks.ConvergenceMetric`
     """
     def __str__(self):
-        """
-        Return a string representation of Metric class
+        """Get a string representation of Metric class.
+
+        Returns
+        -------
+        str
+            Human readable representation of the metric's type.
+
         """
         if self.title is not None:
             return self.title
         else:
             return type(self).__name__[:-6]
 
     def set_parameters(self, **parameters):
-        """
-        Set the parameters
+        """Set the metric's parameters.
+
+        Parameters
+        ----------
+        **parameters
```

Comment:
type missing (for all kwargs)

Reply:
I do this on purpose because the type can be anything in this function.